cfgov/alerts/post_django_errors.py

```diff
@@ -0,0 +1,14 @@
+import logging
```

Comment:
I think the naming of the `post_django_errors.py` file might want to be more consistent with other logging handler module names since it's just a Python logging handler, not Django-specific.